.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
:red_circle: As part of the requirements for your .gitignore file, you should add the node_modules directory (as it can be added by accident while setting up Linters).

Reply:
Hello, I made mistake by merging the first pull. And now it cannot be opened back. So please consider this pull request.
File name can also be  not changed now.

Reply:
The branch name and file name cannot be changed because they require a new pull request.
Also by mistake, I merged the first pull request and later deleted that. So I had to make new pull request for review.